static/app/components/events/eventStatisticalDetector/regressionMessage.tsx

```diff
@@ -62,15 +62,26 @@ function EventStatisticalDetectorRegressedPerformanceMessage({
     <DataSection>
       <div style={{display: 'inline'}}>
         {tct(
-          '[detected] Based on the transaction [transactionName], there was a [amount] increase in duration (P95) around [date] at [time]. Overall operation percentage changes indicate what may have changed in the regression.',
+          'Based on the transaction [transactionName], there was a [amount] increase in duration (P95) from [previousDuration] to [regressionDuration] around [date] at [time]. Overall operation percentage changes indicate what may have changed in the regression.',
           {
-            detected: <strong>{t('Detected:')}</strong>,
             transactionName: (
               <Link to={normalizeUrl(transactionSummaryLink)}>{transactionName}</Link>
             ),
             amount: formatPercentage(
               event?.occurrence?.evidenceData?.trendPercentage - 1
             ),
+            previousDuration: (
+              <PerformanceDuration
+                abbreviation
```

Comment:
@Zylphrex I noticed your issues are using the full unit, i.e. millisecond instead of ms, do you want to use the abbreviation? I can add it here.

Reply:
Let's use the abbreviation 👍